docs/_articles/en/functions/overview.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Overview
+lang: en
+---
+## Salesforce Functions (Beta)
+Use the Salesforce Functions extensions in VSCode to build event-driven, elastically scalable apps and experiences.
+
+Use the Salesforce features for VS Code to create and test Salesforce Functions locally.
```

Comment:
If this sentence is intended to be a lead-in to the head3s, then I would edit it to say something like:
Use these VS Code features to create and test... But I'm not sure this extra sentence adds any real value that the first sentence doesn't already cover.

Reply:
How is this:
Use the Salesforce Functions features in VSCode to build event-driven, elastically scalable apps and experiences. You can create and test functions against sample payloads locally using the VSCode IDE.  @ekapner

Reply:
@sbudhirajadoc  - one small tweak. 

Use the Salesforce Functions features in VSCode to build event-driven, elastically scalable apps and experiences. You can create and test functions against sample payloads locally in VS Code.

Reply:
I think there's a space between VS and Code - I looked at MS docs and they include a space.

Reply:
Got it. Thanks Emily!